src/test/scala/mesosphere/marathon/test/SettableClock.scala

```diff
@@ -24,11 +25,17 @@ class SettableClock(private[this] var clock: Clock = SettableClock.defaultJavaCl
 
   override def withZone(zoneId: ZoneId): Clock = new SettableClock(clock.withZone(zoneId))
 
+  @deprecated("Use advanceBy instead")
   def +=(duration: FiniteDuration): Unit = plus(duration)
 
+  def advanceBy(duration: FiniteDuration): this.type =
+    plus(DurationConverters.toJava(duration))
+
+  @deprecated("Use advanceBy instead")
```

Comment:
🤷‍♂️ 

Do we want our tests to always work with `FiniteDuration`? Do we value the explicitness of `advanceBy` over `+=`? Something else?